src/pages/Home.vue

```diff
@@ -0,0 +1,13 @@
+<template>
```

Comment:
this looks as a demo page.

Reply:
Yes that's the demo page, but the **/** is showing the login form.